kuma/users/jinja2/users/email/welcome/plain.ltxt

```diff
@@ -36,10 +36,10 @@
     {{ _("Join to tell us about what you're interested in on MDN and ask questions about the site.") }}
 
 {# L10n: This is an email. Whitespace matters! #}
-* {{ _('Real-time chat: #mdn channel on irc.mozilla.org') }}
+* {{ _('Real-time chat: #mdn room on chat.mozilla.org') }}
 
 {# L10n: This is an email. Whitespace matters! #}
-    ({{ _('See %(irc_link)s for more info about IRC.', irc_link=add_utm('https://wiki.mozilla.org/IRC', 'welcome')) }})
+    ({{ _('See %(matrix_link)s for more info about Matrix.', matrix_link=add_utm('https://wiki.mozilla.org/Matrix', 'welcome')) }})
```

Comment:
I think it would be nice to have this merged before we tackle https://github.com/mdn/kuma/issues/6883